test/controllers/graphql_controller_v2_test.rb

```diff
@@ -308,18 +309,19 @@ def setup
     )
   end
 
-  test 'graphqlv2 geobox search longitude order matters' do
+  test 'graphql geobox search longitude order matters' do
     # This is fragile to our collection having an equal number of records in both hemispheres.
     eastern_hits = 0
     western_hits = 0
-    VCR.use_cassette('graphqlv2 geobox eastern hemisphere') do
+    VCR.use_cassette('graphql geobox eastern hemisphere') do
       post '/graphql', params: { query: '{
                                   search(geobox: {
                                     minLongitude: 0,
                                     minLatitude: -90,
                                     maxLongitude: 180,
                                     maxLatitude: 90
-                                  }) {
+                                  },
+                                  source: "MIT GIS Resources") {
```

Comment:
I think I understand why we're adding source parameters to this search, because now the tests are running against a bigger index, so a source filter is needed to get down to only GIS records. But if that is the case, the same parameter should also be added to the cassette on line 343 to keep them in sync. Otherwise the test for different results on line 369 can be down to different source values, rather than different geobox definitions.

Reply:
Good catch. I thought I had done that but clearly didn't. I'll fix.

The reason I am making this change is solely because the test failed with the larger index because the hits for both queries came back as 10,000 so I limited to just MIT data... poorly :)

Will fix.

Reply:
I've adjusted this test to have both hemispheres use the same source filter. I've also reconfigured it to only use a single cassette for the whole test as we can instruct the cassette to have responses for all of the queries that take place during that test which is generally a bit easier to maintain.